tenable/io/exports.py

```diff
@@ -12,15 +12,17 @@
 
     .. automethod:: assets
     .. automethod:: vulns
+    .. automethod:: compliance
 '''
-from .base import TIOEndpoint, APIResultsIterator, UnexpectedValueError
-from tenable.errors import TioExportsError, TioExportsTimeout
+import time
+import sys
 from ipaddress import IPv4Network
+from tenable.errors import TioExportsError, TioExportsTimeout
+from .base import TIOEndpoint, APIResultsIterator, UnexpectedValueError
 try:
     from json.decoder import JSONDecodeError
 except ImportError:
     JSONDecodeError = ValueError
-import time, sys
 
 class ExportsIterator(APIResultsIterator):
     '''
```

Comment:
Export iterator needs to have a value as counter

Reply:
Iterator already have attribute count to return count of fetched records.
updated docstring for attribute details